File: SistemaDecomisos/src/java/model/Lugar.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package model;

import java.io.Serializable;

/**
 *
 * @author Marco
 */
public class Lugar implements Serializable, Jsonable{
    private Distrito distrito;
    private String direccionExacta;

    public Lugar() {
    }

    public Lugar(Distrito distrito, String direccionExacta) {
        this.distrito = distrito;
        this.direccionExacta = direccionExacta;
    }

    public Distrito getDistrito() {
        return distrito;
    }

    public void setDistrito(Distrito distrito) {
        this.distrito = distrito;
    }

    public String getDireccionExacta() {
        return direccionExacta;
    }

    public void setDireccionExacta(String direccionExacta) {
        this.direccionExacta = direccionExacta;
    }
    
    
}

File: web/js/objects/Lugar.js
function Lugar(distrito, direccionExacta) {
    this.Lugar(distrito, direccionExacta);
}

Lugar.prototype = {
    distrito : "",
    direccionExacta: "",
    Lugar: function (distrito, direccionExacta) {
        this.distrito = distrito, 
        this.direccionExacta = direccionExacta;
    }
};

Lugar.from = function (plain) {
    lugar = new Lugar(
        plain.distrito, 
        plain.direccionExacta
    );
    return lugar;
};

Lugar.to = function (lugar) {
    return {
        _class: 'Lugar',
        distrito: lugar.distrito, 
        direccionExacta : lugar.direccionExacta
    };
};
 